doc/releasenotes.html

```diff
@@ -78,6 +78,114 @@ <h2>CATEGORY FOR MAJOR RELEASES: General, Solar, Detailed Photovoltaic, Wind Pow
+
+<p>Correct battery state of charge (SOC) calculation at the end of a grid outage period. (<a href="https://github.com/NREL/ssc/pull/1278">SAM 1278</a>)</p>
+
+<p>Improve behavior of stacked-area plots on Time Series tab of Results page. (<a href="https://github.com/NREL/SAM/pull/1987">SAM 1987</a>)</p>
+
+<p>Fix numerical error in Li-ion NMC/Graphite battery lifetime code that caused unexpected annual charge energy values of zero. (<a href="https://github.com/NREL/ssc/pull/1310">SAM 1310</a>)</p>
+
+<p>Add support for exporting battery power to the grid for BTM retail rates dispatch option. (<a href="https://github.com/NREL/SAM/pull/2041">SAM 2041</a>, <a href="https://github.com/NREL/ssc/pull/1303")>SSC 1303</a>)</p>
+
+<p>Fix typo in internal 'batt_adjust_constant' variable name. (<a href="https://github.com/NREL/ssc/pull/1301">SAM 1301</a>)
+
+<p>Improve error handling of forecast price signals object to prevent SSC crashes for SAM Software Development Kit (SDK), PySAM, and gtest. (<a href="https://github.com/NREL/ssc/pull/1288">SAM 1288</a>)</p>
+
+<h2>Marine Energy (SAM 2025.4.16)</h2>
+
+<p>Update cost curves. (<a href="https://github.com/NREL/SAM/pull/2043">SAM 2043</a>)</p>
+
+<p>Fix conversion of time-series wave resource data to joint probability distribution (JPD) for files with three-hour data for over three years. (<a href="https://github.com/NREL/SAM/pull/2043">SAM 2043</a>)</p>
+
+<p>Add cost curve for RM2 wave energy converter. (<a href="https://github.com/NREL/ssc/pull/1305">SSC 1305</a>)
+
+<h2>Wind Power (SAM 2025.4.16)</h2>
+
+<p>Fix double-counting of time series adjustment factors. (<a href="https://github.com/NREL/ssc/pull/1284">SAM 1284</a>)</p>
+
+<h2>Custom Generation Profile (SAM 2025.4.16)</h2>
+
+<p>Fix problem with combining generation profiles from cases with different weather file time steps. (<a href="https://github.com/NREL/SAM/pull/2012">SAM 2012</a>)</p>
+
+<h2>Financial Models (SAM 2025.4.16)</h2>
+
+<p>Provide feedback for behind-the-meter (BTM) with billing demand enabled when "Included in Billing Demand" is zero for all periods. (<a href="https://github.com/NREL/ssc/pull/1293">SSC 1293</a>)</p>
+
+<p>Remove duplicate electricity bill output variables. (<a href="https://github.com/NREL/SAM/pull/2009">SAM 2009</a>, <a href="https://github.com/NREL/ssc/pull/1276">SSC 1276</a>)</p>
+
+<p>Add missing fuel cell O&M cost line item to cashflow and fix Send-to-Excel with Equations callback error for Fuel Cell / Commercial configuration. (<a href="https://github.com/NREL/SAM/pull/1995">SAM 1995</a>)</p>
+
+<h2>Hybrid Systems (SAM 2025.4.16)</h2>
+
+<p>Fix "Apply values from library" button for time-of-delivery (TOD) factors on Revenue input page. (<a href="https://github.com/NREL/SAM/pull/1999">SAM 1999</a>)</p>
+
 <h1 id="2024-12-12">SAM 2024.12.12, SSC 298: December 12, 2024</h1>
```

Comment:
Does this line remain for the IPH models?

Reply:
>Does this line remain for the IPH models?

I just fixed an open parenthesis in the IPH section -- is that what you were referring to?